### ISSUE TITLE
Staging Review finding: Screen reader users may miss "Time-sensitive" label

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98568]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the results page, benefits that are time-sensitive have a "Time-sensitive benefit" label at the top of the card. However, a common interaction pattern for screen reader users exploring page content is to skip heading-by-heading through the page, scanning the page to find sections they're interested in exploring in more detail. Since the "time-sensitive" label appears before the heading, users exploring the page in this way are likely to skip over it as they jump to the heading for that card.

<details><summary>Link, screenshot or steps to recreate</summary>

Example of a time-sensitive card:
```
<va-card tabindex="0" class="hydrated">
   <div class="blue-heading">
      <span>
         <b>Time-sensitive benefit</b>
      </span>
   </div>
   <h3 class="vads-u-margin-top--0">
      <span class="usa-label category-label">Disability</span>
      <br><br>
      <span>Disability compensation</span>
   </h3>
   <p class="vads-u-margin-y--0">[content]</p>
   [links]
</va-card>
```

</details>

## Recommended action

There are two approaches here that could make sense.

Option 1 - wrap the "time-sensitive" label inside the heading along with the heading text and category label, so it's announced as a single element (eg. "heading level 3, Time-sensitive benefit, Disability, Disability compensation"):

Option 2 - associate the "time-sensitive" label with the heading using `aria-describedby`. This is a good use of `aria-describedby` as a tool for providing additional information about an element, eg:
```
   <div id="some-id" class="blue-heading">
      <span>
         <b>Time-sensitive benefit</b>
      </span>
   </div>
   <h3 class="vads-u-margin-top--0" aria-describedby="some-id">
      <span class="usa-label category-label">Disability</span>
      <br><br>
      <span>Disability compensation</span>
   </h3>
```

This would be announced as "heading level 3, Disability, Disability compensation, Time-sensitive benefit."

Either approach would be fine here!

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
